package.json

```diff
@@ -5,7 +5,8 @@
   "private": true,
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack -p",
-    "start": "webpack-dev-server --mode=development"
+    "start": "webpack-dev-server --mode=development",
+    "eslint": "node_modules/.bin/eslint src/"
```

Comment:
there is no need to write path inside npm scripts - so it will be just 'eslint src/'